gems/pending/appliance_console/database_configuration.rb

```diff
@@ -79,7 +79,6 @@ def activate
     end
 
     def post_activation
-      ServiceGroup.new.restart_services
       LinuxAdmin::Service.new("evmserverd").enable.start
```

Comment:
👍 Seems like the right thing to do now that systemd can handle all of this for us.  

Are all the other services enabled and running (besides the custom logic here for evmserverd?
`SERVICES  = %w(evminit memcached miqtop evmserverd).freeze`